qlib/utils/serial.py

```
@@ -146,8 +147,12 @@ def load(cls, filepath):
         Returns:
             `type(cls)`: the instance of `type(cls)`
         """
+        logger = get_module_logger("serial")
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)